pkg/agent/datamodel/types.go

```diff
@@ -544,6 +544,9 @@ type CustomKubeletConfig struct {
 	TopologyManagerPolicy string    `json:"topologyManagerPolicy,omitempty"`
 	AllowedUnsafeSysctls  *[]string `json:"allowedUnsafeSysctls,omitempty"`
 	FailSwapOn            *bool     `json:"failSwapOn,omitempty"`
+	ContainerLogMaxSizeMB *int32    `json:"containerLogMaxSizeMB,omitempty"`
+	ContainerLogMaxFiles  *int32    `json:"containerLogMaxFiles,omitempty"`
+	PodMaxPids            *int32    `json:"podMaxPids,omitempty"`
```

Comment:
nit -- might be more clear to rename the same field on the other struct, PodPidsLimit

Reply:
Yes it should be PodPidsLimit but it is hard to update API, I'll leave as it is for now.